src/commands/deploy.js

```diff
@@ -68,12 +68,12 @@ deploy.restart = function(api, params) {
   });
 
   var s_deploy = s_appData.flatMapLatest(function(app_data) {
-    var suffix = "";
-    if(commitId) suffix += " on commit #" + commitId;
-    if(withoutCache) suffix += " without using cache";
-
-    Logger.println("Restarting " + app_data.name + suffix);
-    return Application.redeploy(api, app_data.app_id, app_data.org_id, commitId, withoutCache);
+    return s_commitId.flatMapLatest((theCommitId) => {
+      var suffix = " on commit #" + (commitId || theCommitId);
```

Comment:
can you use `let` instead?

Reply:
I'll remove it when I'll rebase on master.